src/config/server.rs

```diff
@@ -20,6 +22,9 @@ pub struct Server {
   pub port: Option<u16>,
   pub proxy: Option<Proxy>,
   pub vars: Option<BTreeMap<String, String>>,
+  pub add_response_headers: Option<Vec<(String, String)>>,
```

Comment:
```suggestion
  pub add_response_headers: KeyValues,
```

Reply:
@tusharmath  If we replace Option<Vec<(String, String)>> by Key values which is ultimately pub BTreeMap<String, String>, it wil give error when header conatins - (hypen) character.